src/main/java/com/bulkbuy/enterprise/controller/NavigationController.java

```diff
@@ -30,6 +32,7 @@ public String getAbout(Model model){
     @GetMapping("/placeOrder")
     public String getPlaceOrder(Model model){
         model.addAttribute("activePage", "placeOrder");
+        model.addAttribute("saveOrder", new Order());
```

Comment:
Added the Order object as an attribute to the /placeOrder endpoint that returns the placeOrder.html to bind the object to the page. This allows the page to bind the fields of the object to the elements in the forms.